pkg/alertmanager/state_replication.go

```diff
@@ -154,14 +224,21 @@ func (s *state) running(ctx context.Context) error {
 	}
 }
 
+func (s *state) broadcast(key string, b []byte) {
+	// We should ignore the Merges into the initial state during settling.
+	if s.Ready() {
```

Comment:
Do we need this? IIRC, we decided to go with a no rebroadcasting of state on calls to `Merge`  under that scheme this would never occur unless we received a request directly to this Alertmanager

Reply:
Good spot - yes we do need it for now, until the re-broadcasting is disabled. I wanted to do it before this commit, but it seems that disabling it is not trivial as we thought and will involve upstream changes.